parsl/dataflow/dflow.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, config=Config()):
                     self.workflow_name = fname
                     break
 
-        self.workflow_version = str(self.time_began)
+        self.workflow_version = str(self.time_began.replace(microsecond=0))
```

Comment:
This information doesn't need destroying if it gets dropped in the UI later (which it does) ?

Reply:
This `workflow_version` is a customizable variable in `monitoring` -- users can input something like a string. Thus, this value is stored in the database as `Text`, rather than `float` or `datetime` type.

I think we do not want to drop microsecond at the UI since we have no idea this is a customized string or a datetime string. So, I choose to destroy before it is commited to database.

Reply:
ok